### DISCUSSION TITLE
Is the local agent's control socket documented somewhere ?

### DISCUSSION THREAD

**Discussion by TwilCynder** [alumet-dev/alumet#72] [Q&A] [answered]
I haven't been able to find a documentation for the different commands that can be sent to the control socket, is there one anywhere ? And if not, where should I look to add one ?

**Comment by TheElectronWill** [comment]
There is a small documentation [in the rustdoc of the plugin-socket-control](https://github.com/alumet-dev/alumet/blob/main/plugin-socket-control/src/command.rs#L37), and an example [in the readme of the socket-control plugin](https://github.com/alumet-dev/alumet/tree/main/plugin-socket-control).

But ideally, it should be added to the [user book](https://github.com/alumet-dev/user-book) :slightly_smiling_face:

**Comment by titouanj** [comment, marked as answer]
Hi, a real readme has been pushed for the socket-control plugin in #252 
You can find this readme here: https://github.com/alumet-dev/alumet/tree/main/plugins/socket-control#readme
